node/src/main/scala/pravda/node/data/PravdaConfig.scala

```diff
@@ -46,12 +45,12 @@ object PravdaConfig {
 
   final case class GenesisValidator(
       publicKey: CryptoKey,
-      power: Int,
+      power: String,
```

Comment:
Why the type of the votePower was changed to `String`? It has `int64` type in the tendermint core.

Reply:
Yes, but for JSON need quotes.
`ERROR: Failed to create node: invalid character -- Amino:JSON int/int64/uint/uint64 expects quoted values for javascript numeric support`